"Can't adapt type 'Instance'" error raised in species list API endpoint

`ProgrammingError: can't adapt type 'Instance'`

```
2015-02-27 09:33:49,721 [ERROR] django.request: Internal Server Error: /api/v3/instance/phillytreemap/species
Traceback (most recent call last):
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 57, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/otm/app/opentreemap/api/decorators.py", line 107, in wrapperf
    return view_f(request, *args, **kwargs)
  File "/usr/local/otm/app/opentreemap/api/decorators.py", line 151, in wrapper
    return view_f(request, *args, **kwargs)
  File "/usr/local/otm/app/opentreemap/treemap/decorators.py", line 45, in wrapper
    return view_fn(request, instance, *args, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django_tinsel/decorators.py", line 125, in newreq
    outp = req_function(request, *args, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django_tinsel/decorators.py", line 106, in newreq
    outp = req_function(request, *args, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django_tinsel/decorators.py", line 33, in routed
    return req_method(request, *args2, **kwargs2)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django_tinsel/decorators.py", line 125, in newreq
    outp = req_function(request, *args, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django_tinsel/decorators.py", line 106, in newreq
    outp = req_function(request, *args, **kwargs)
  File "/usr/local/otm/app/opentreemap/treemap/decorators.py", line 30, in wrapper
    instance = get_instance_or_404(url_name=instance_url_name)
  File "/usr/local/otm/app/opentreemap/treemap/util.py", line 115, in get_instance_or_404
    return get_object_or_404(Instance, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 113, in get_object_or_404
    return queryset.get(*args, **kwargs)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 301, in get
    num = len(clone)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 77, in __len__
    self._fetch_all()
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 854, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/query.py", line 220, in iterator
    for row in compiler.results_iter():
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 710, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 781, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 53, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/utils.py", line 99, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/otm/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 53, in execute
    return self.cursor.execute(sql, params)
ProgrammingError: can't adapt type 'Instance'

--- a/opentreemap/api/tests.py
+++ b/opentreemap/api/tests.py
@@ -1773,3 +1773,17 @@ class PasswordResetTest(OTMTestCase):
         response = post_json(url % (API_PFX, self.jim.email),
                              {}, self.client, None)
         self.assertEquals(response.status_code, 200)
+
+
+class SpeciesListTest(OTMTestCase):
+    def setUp(self):
+        self.instance = setupTreemapEnv()
+        self.user = User.objects.get(username="jim")
+
+    def test_species_list_endpoint(self):
+        response = get_signed(
+            self.client,
+            "%s/instance/%s/species" % (API_PFX,
+                                        self.instance.url_name),
+            user=self.user)
+        self.assertEqual(response.status_code, 200)

--- a/opentreemap/api/views.py
+++ b/opentreemap/api/views.py
@@ -19,7 +19,7 @@ from django_tinsel.utils import decorate as do
 from django_tinsel.decorators import route, json_api_call
 
 from treemap.models import Plot, Tree, User
-from treemap.routes import species_list
+from treemap.views.misc import species_list
 from treemap.lib.map_feature import context_dict_for_plot
 from treemap.lib.tree import add_tree_photo_helper
 from treemap.lib.photo import context_dict_for_photo

